### DISCUSSION TITLE
Blackfire profiling cli with --env

### DISCUSSION THREAD

**Discussion by manuelson** [wardenenv/warden#697] [Ideas]
When you launch the command **warden blackfire run {command}** blackfire gives you the option to add the --env parameter.

Warden should support a new global variable on .env file (e.g. **BLACKFIRE_ENV_ID**) by editing the .cmd **commands/blackfire.cmd** it should accept the **--env** parameter as documented by blackfire (https://blackfire.io/docs/profiling-cookbooks/profiling-cli)

**Comment by navarr** [comment]
This looks like a very straightforward feature to add.  I've created an issue for your feature request https://github.com/wardenenv/warden/issues/699
